examples/solids/qfunctions/hyperFS.h

```diff
@@ -0,0 +1,387 @@
+// Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-734707. All Rights
+// reserved. See files LICENSE and NOTICE for details.
+//
+// This file is part of CEED, a collection of benchmarks, miniapps, software
+// libraries and APIs for efficient high-order finite element and spectral
+// element discretizations for exascale applications. For more information and
+// source code availability see http://github.com/ceed.
+//
+// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
+// a collaborative effort of two U.S. Department of Energy organizations (Office
+// of Science and the National Nuclear Security Administration) responsible for
+// the planning and preparation of a capable exascale ecosystem, including
+// software, applications, hardware, advanced system engineering and early
+// testbed platforms, in support of the nation's exascale computing imperative.
+
+#ifndef HYPER_FS_H
+#define HYPER_FS_H
+
+#ifndef __CUDACC__
+#  include <math.h>
+#endif
+
+#ifndef PHYSICS_STRUCT
+#define PHYSICS_STRUCT
+typedef struct Physics_private *Physics;
+struct Physics_private {
+  CeedScalar   nu;      // Poisson's ratio
+  CeedScalar   E;       // Young's Modulus
+};
+#endif
+
+// -----------------------------------------------------------------------------
```

Comment:
I know that now we have an awesome extensive documentation, but shouldn't these functions at least have a short help docstring? Not all people might recollect at a first glance what your acronyms mean. All other files (like in the NS example) have the `@file`

Reply:
Thanks for catching that